### ISSUE TITLE
Some tests failing with python 3.6 and 3.7

### ISSUE THREAD

**Issue by nyanloutre** [lincolnloop/python-qrcode#163]
Hello,

I noticed that tests don't pass on python 3.6 and 3.7 with the following error:

```
======================================================================
FAIL: test_invalid_data (qrcode.tests.test_release.UpdateManpageTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nix/store/zls48cbfg36i19pvn1gvprxyvwyginhk-python3-3.7.1/lib/python3.7/unittest/mock.py", line 1191, in patched
    return func(*args, **keywargs)
  File "/build/qrcode-6.0/qrcode/tests/test_release.py", line 20, in test_invalid_data
    mock_file.assert_called()
  File "/nix/store/zls48cbfg36i19pvn1gvprxyvwyginhk-python3-3.7.1/lib/python3.7/unittest/mock.py", line 792, in assert_called
    raise AssertionError(msg)
AssertionError: Expected 'open' to have been called.

======================================================================
FAIL: test_no_change (qrcode.tests.test_release.UpdateManpageTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nix/store/zls48cbfg36i19pvn1gvprxyvwyginhk-python3-3.7.1/lib/python3.7/unittest/mock.py", line 1191, in patched
    return func(*args, **keywargs)
  File "/build/qrcode-6.0/qrcode/tests/test_release.py", line 31, in test_no_change
    mock_file.assert_called()
  File "/nix/store/zls48cbfg36i19pvn1gvprxyvwyginhk-python3-3.7.1/lib/python3.7/unittest/mock.py", line 792, in assert_called
    raise AssertionError(msg)
AssertionError: Expected 'open' to have been called.
```

It could be related to this line: https://github.com/lincolnloop/python-qrcode/blob/35056062270a3d5084b1217df611d09cd94c5ca3/qrcode/tests/test_release.py#L18

**Comment by SmileyChris** [comment]
Fixed in 35056062270a3d5084b1217df611d09cd94c5ca3, just hadn't rolled a new release. Will do that now.

**Comment by nyanloutre** [comment]
Thanks !
